src/config/datamodel/board.py

```diff
@@ -11,8 +11,14 @@
 
 
 import attr
+import datetime
+from datetime import timezone
 
 
 @attr.dataclass
 class BoardConfigRoot:
     cycle_at: int = attr.ib(validator=attr.validators.instance_of(int))
+    datetime_last_case: datetime.datetime = attr.ib(
```

Comment:
[question] Please justify why this would be beneficial to be configurable?

Reply:
It isn't, I was drawn here because I was told that the datamodel had to be expanded, and this seemed logical to do so.